### DISCUSSION TITLE
[Question]I use labelme in Win10, when I want delete a point, the edit label and detel polygon is gray which means can not choice. How can I to sovel it?

### DISCUSSION THREAD

**Discussion by vastyao** [wkentaro/labelme#1072] [Q&A / Help]
I use labelme in Win10, when I want delete a point, the edit label and detel polygon is gray which means can not choice. How can I to sovel it?

**Comment by 307321587** [comment]
You can right-click on the selected point and delete it via the delete selected point option in the context menu.  Holding the shift button and clicking the point also works

**Comment by vastyao** [comment]
I know your means, but that cion can not been chosen

**Comment by vastyao** [comment]
Anyway, Thank you the same.
